### PR TITLE
More dailylimits work

### PR DESCRIPTION
**Pull request by gausie**
Some cleaning up, more test coverage, more cases handled, more edge cases handled

### CODE REVIEW

File: src/data/dailylimits.txt [hunk 1]
```diff
@@ -1,6 +1,124 @@
 1
 # Limiter: daily limit
 # Type	Item name	Tracking property	[Max = 1]
+Cast	Acquire Rhinestones	_rhinestonesAcquired
```
Comment (jaadams5) [inline src/data/dailylimits.txt:4]: Sorting is better than not sorting.  Are we pretty much convinced that we want case sensitive sorts in code where a string sort is used because there is no associated numeric?  Just asking for future reference.

**Review by jaadams5 — APPROVED** [review]

File: src/data/dailylimits.txt (hunk 1 above)
Comment (gausie) [reply src/data/dailylimits.txt:4]: I just used IntelliJ's sort function - I have no real opinion on that just yet
